web-ui/src/components/certifications/CertificationBadges.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
```

Comment:
This component just displays a row of badge images for the certifications that have been earned by a given member.